demo/src/main/java/com/novoda/merlin/demo/presentation/base/MerlinActivity.java

```diff
@@ -50,6 +50,12 @@ protected void onStop() {
         Logger.detach(logHandle);
     }
 
+    @Override
+    protected void onDestroy() {
+        merlin = null;
```

Comment:
Is there a reason why we're not doing that now instead? It seems a big enough concern for us to address it here.